### ISSUE TITLE
OpenCV Linker error with nightly on VS

### ISSUE THREAD

**Issue by ofTheo** [openframeworks/openFrameworks#4981]
Not sure why yet - but a lot of errors relating to OpenCv 

![buildcv](https://cloud.githubusercontent.com/assets/144000/13650018/a675605c-e60e-11e5-96e3-c99b31e9a5f1.jpg)

**Comment by arturoc** [comment]
yes, somebody has reported this already, i think in the forum only. it seems like it would be trying to use ipp so it's probably just some missing/extra flag in the apothecary config.

**Comment by prisonerjohn** [comment]
I just ran into this issue too. I'm not seeing any mention of opencv in apothecary, am I looking in the right place? https://github.com/openframeworks/openFrameworks/tree/master/scripts/apothecary/formulas

**Comment by arturoc** [comment]
the most up to date version of apothecary is in https://github.com/openframeworks//apothecary but that doesn't contain addons yet, the addons formulas are in the addons folder of this repo. you can copy the opencv formula from ofxOpenCv to the apothecary repo and build it there.
